assets/src/js/gravityforms-client-side.js

```diff
@@ -1,13 +1,24 @@
+/* global dataLayer */
+
 (() => {
-  if (typeof p4GfClientSideConfig !== 'undefined') {
-    const urlParams = new URLSearchParams(window.location.search);
-    // eslint-disable-next-line no-undef
-    p4GfClientSideConfig.populate.forEach(field => {
-      const value = urlParams.get(field.parameter);
-
-      if (value !== null) {
-        document.getElementById(field.fieldId).value = value;
-      }
-    });
+  if (typeof p4GfClientSideConfig === 'undefined') {
+    return;
   }
```

Comment:
Should we also check for `dataLayer` here?

```suggestion
  if (typeof p4GfClientSideConfig === 'undefined') {
    return;
  }
  if (typeof dataLayer === 'undefined') {
    return;
  }
```

Reply:
No, because it's added at the top of the file to be used as global